src/Client.php

```diff
@@ -326,7 +326,25 @@ public function generatePrivateChannelToken($client, $channel, $exp = 0, $info =
         return implode('.', $segments);
     }
 
-    private function send($method, $params = [])
+/*
+ * Function added for backward compatibility with PHP version < 5.5
+ */
+    
+    private function json_last_error_msg() {
```

Comment:
Do I understand right that on PHP >5.4 will use this function too instead of native one?